packages/zwave-js/src/lib/commandclass/SceneActuatorConfigurationCC.ts

```diff
@@ -0,0 +1,395 @@
+			SceneActuatorConfigurationCommand.Get,
+		);
+
+		const cc = new SceneActuatorConfigurationCCGet(this.driver, {
+			nodeId: this.endpoint.nodeId,
+			endpoint: this.endpoint.index,
+			sceneId: sceneId,
+		});
+		const response = await this.driver.sendCommand<SceneActuatorConfigurationCCReport>(
+			cc,
+			this.commandOptions,
+		);
+
+		if (response) {
+			return pick(response, ["sceneId", "level", "dimmingDuration"]);
+		}
+	}
+}
+
+@commandClass(CommandClasses["Scene Actuator Configuration"])
+@implementedVersion(1)
+export class SceneActuatorConfigurationCC extends CommandClass {
+	declare ccCommand: SceneActuatorConfigurationCommand;
+
+	// interview is omitted, because our only option would be
+	// to query all 255 possible sceneIds, which is a lot of traffic
+	// for little benefit
+}
+
+interface SceneActuatorConfigurationCCSetOptions extends CCCommandOptions {
+	sceneId: number;
+	dimmingDuration: Duration;
+	level?: number;
+}
+
+@CCCommand(SceneActuatorConfigurationCommand.Set)
+export class SceneActuatorConfigurationCCSet extends SceneActuatorConfigurationCC {
+	public constructor(
+		driver: Driver,
+		options:
+			| CommandClassDeserializationOptions
+			| SceneActuatorConfigurationCCSetOptions,
+	) {
+		super(driver, options);
+		if (gotDeserializationOptions(options)) {
+			// TODO: Deserialize payload
+			throw new ZWaveError(
+				`${this.constructor.name}: deserialization not implemented`,
+				ZWaveErrorCodes.Deserialization_NotImplemented,
+			);
+		} else {
+			if (options.sceneId < 1 || options.sceneId > 255) {
+				throw new ZWaveError(
+					`scene id ${options.sceneId} out of range [1..255]`,
+					ZWaveErrorCodes.Argument_Invalid,
+				);
+			}
+			this.sceneId = options.sceneId;
+			this.dimmingDuration = options.dimmingDuration;
+			if (options.level != undefined) {
+				if (options.level < 0 || options.level > 255) {
+					throw new ZWaveError(
+						`level ${options.level} out of range [1..255]`,
+						ZWaveErrorCodes.Argument_Invalid,
+					);
+				}
+				this.override = true;
+				this.level = options.level;
+			} else {
+				this.override = false;
+				this.level = 0xff; // will be ignored
+			}
+		}
+	}
+
+	public sceneId: number;
+	public dimmingDuration: Duration;
+	public override: boolean;
+	public level: number;
+
+	public serialize(): Buffer {
+		this.payload = Buffer.from([
+			this.sceneId,
+			this.dimmingDuration.serializeSet(),
+			this.override ? 0b1000_0000 : 0x00,
+			this.level,
+		]);
+		return super.serialize();
+	}
+}
+
+@CCCommand(SceneActuatorConfigurationCommand.Report)
+export class SceneActuatorConfigurationCCReport extends SceneActuatorConfigurationCC {
+	public constructor(
+		driver: Driver,
+		options: CommandClassDeserializationOptions,
+	) {
+		super(driver, options);
+		validatePayload(this.payload.length >= 3);
+		this.sceneId = this.payload[0];
+		this.level = this.payload[1];
+		this.dimmingDuration =
+			Duration.parseReport(this.payload[2]) ?? new Duration(0, "unknown");
```

Comment:
> The value 0 MUST indicate that no scene is currently active at the sending node and no scene setting is
advertised in this command. In this case, the Level and Dimming Duration fields SHOULD be ignored.

I think it would make sense to keep level and dimming duration as optional. What do you think?

Reply:
Yes, the `level` and `dimmingDuration` fields should be undefined in a report for `sceneId = 0`.  That also means we should not persist values if `sceneId = 0`.

I'm changing the constructor to only call `persistValues()` if `sceneId > 0`, but `persistValues` is a `public` function, so I assume I have to consider the possibility that the function is called from outside the constructor.  Is it correct to have `persistValues` abort and `return false` if it's called when `sceneId = 0`?

Reply:
> Is it correct to have persistValues abort and return false if it's called when sceneId = 0?

The return type of the subclassed method is currently not evaluated anywhere, but for consistency: **yes**!